cli/CampaignConfigValidation/LoggingCampaignConfigurationLoader.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Cli\CampaignConfigValidation;
+
+use Symfony\Component\Yaml\Exception\ParseException;
+use WMDE\Fundraising\Frontend\BucketTesting\CampaignConfigurationLoaderInterface;
+
+class LoggingCampaignConfigurationLoader implements CampaignConfigurationLoaderInterface {
+
+	private $actualCampaignConfigurationLoader;
+	private $errorLogger;
+
+	public function __construct( CampaignConfigurationLoaderInterface $campaignConfigurationLoader, ValidationErrorLogger $errorLogger ) {
+		$this->errorLogger = $errorLogger;
+		$this->actualCampaignConfigurationLoader = $campaignConfigurationLoader;
+	}
+
+	public function loadCampaignConfiguration( string ...$configFiles ): array {
+		try {
+			return $this->actualCampaignConfigurationLoader->loadCampaignConfiguration( ...$configFiles );
+		}
+		catch ( ParseException $e ) {
+			$this->errorLogger->addError( 'Failed to parse YAML: ' . $e->getMessage() );
+		}
+		catch ( \LogicException $e ) {
+			$this->errorLogger->addError( 'Logical error in YAML: ' . $e->getMessage() );
+		}
+		catch ( \RuntimeException $e ) {
```

Comment:
Two Exceptions are missing: `FileFetchingException` (if a file can't be read for some reason, e.g. permissions) and `InvalidConfigurationException` (if the configuration structure is wrong, e.g. required keys missing, additional or misspelled keys)